src/v/coproc/wasm_event.h

```diff
@@ -25,7 +25,10 @@
 namespace coproc::wasm {
 
 /// Enumerations for all possible values for keys within the record headers
-enum class event_header { action, description, checksum };
+enum class event_header { action, description, checksum, type };
+
+/// Enumerations for all possible coproc_type
+enum class coproc_type { nodejs, data_policy };
```

Comment:
what `data_policy` means ?

Reply:
It is new feature. https://docs.google.com/document/d/1BjNJLQZ3DrgMSMgc0nnCzBxEi5Af26KzpB_Ox5DR-xE/edit?usp=sharing

We need to store js code for data-policy. Because we will use it later for init v8 engine